fixtures/gists-app/app/root.jsx

```diff
@@ -66,7 +66,7 @@ export default function Root() {
         <div data-test-id="content" id="content">
           <Outlet />
         </div>
-        {data.enableScripts && (
+        {data?.enableScripts && (
```

Comment:
This seems unrelated, but I was getting an error reading `enableScripts` of undefined.